runtime/gc_vlhgc/CopyScanCacheVLHGC.hpp

```diff
@@ -52,7 +52,7 @@ class MM_CopyScanCacheVLHGC : public MM_CopyScanCache
 	UDATA _objectSize;   /**< sum of objects sizes copied to this copy cache */
 	U_64 _lowerAgeBound; /**< lowest possible age of any object in this copy cache */
 	U_64 _upperAgeBound; /**< highest possible age of any object in this copy cache */
-	UDATA _arraySplitIndex; /**< The index within the array in scanCurrent to start scanning from (meaningful is J9VM_MODRON_SCAVENGER_CACHE_TYPE_SPLIT_ARRAY is set) */ 
+	UDATA _arraySplitIndex; /**< The index within the array in scanCurrent to start scanning from (meaningful is OMR_SCAVENGER_CACHE_TYPE_SPLIT_ARRAY is set) */
```

Comment:
While touching this, I recommend to replace UDATA with uintptr_t, and similar ones, too. Base MM_CopyScanCache already uses uintptr_t.

Reply:
same for other smaller files you touched